src/modules/xplorers-ai/xplorersAIChatBox.tsx

```
@@ -0,0 +1,42 @@
+            <div key={index} className="">
+              {message?.role === "user" && message?.content !== "" ? (
+                <div className="rounded bg-blue-200 p-2">
+                  {message?.content}
+                </div>
+              ) : message?.role === "system" && message?.content !== "" ? (
+                <div className="rounded bg-white p-2">
+                  {message?.content || "No content provided"}
+                </div>
+              ) : null}
+            </div>
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)